backend/core/src/main/java/org/sonarsource/sonarlint/core/TextSearchIndex.java

```diff
@@ -35,12 +35,12 @@
 /**
  * Indexes text associated to objects, and performs full text search to find matching objects.
  * It is a positional index, so it supports queries consisted of multiple terms, in which case it will find partial term matches in sequence (distance = 1).
- * The result is sorted by score. The score of each term matches is the ratio of the term matches (1 for exact match), 
+ * The result is sorted by score. The score of each term matches is the ratio of the term matches (1 for exact match),
  * and the global score is the sum of the term's scores in the object divided by the total term frequency in the object.
- * 
+ * <p>
  * The generic type should properly implement equals and hashCode.
- * <b>An object cannot be indexed twice</b>. 
- * 
+ * <b>An object cannot be indexed twice</b>.
+ * <p>
```

Comment:
Should this be a closing tag?

Reply:
This is valid HTML 4 :) but fair enough, I will change it to HTML 5